src/documents/CoreAPI.md

```diff
@@ -567,13 +567,13 @@ accepted from external users.
 
 ### fluid.log([logLevel, arg1, ... argn])
 
-* `logLevel {LogLevel}` [optional] One of the members of the `evel` structure or in general some `fluid.marker` with a numeric value holding a priority
+* `logLevel {LogLevel}` [optional] One of the members of the `fluid.logLevel` structure or in general some `fluid.marker` with a numeric value holding a priority
 * `arg1 ... argn {Stringable}` A sequence of arguments to be logged. By default these will be handed unchanged to the multiple arguments of `console.log`. It is wise to convert these
   to `String` yourself (e.g. by `JSON.stringify`) if i) this is safe, and ii) you would not be satisfied by the environment's rendering.
 
 Log a message to a suitable environmental console (in current implementations, `console.log`). If the first argument to `fluid.log` is
-one of the members of the [`evel`](#fluidloglevel) structure, this will be taken as the priority of the logged message - else it
-will default to [`evel.INFO`](#fluidloglevel). If the logged message priority does not exceed that set by the most recent call to the [`fluid.setLogging`](#fluidsetloggingloglevel) function,
+one of the members of the [`fluid.logLevel`](#fluidloglevel) structure, this will be taken as the priority of the logged message - else it
```

Comment:
Not your change but we may as well fix it: `#fluidloglevel` should be `#fluid-loglevel`.

Reply:
See below, this is down to the change from "marked" to "markdown-it" and is to be expected from now on.